docs/program/apis/cloud.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Managing Your Fleet with Viam's Cloud API"
+linkTitle: "Cloud Management"
+weight: 20
+type: "docs"
+description: "How to use the cloud API with Viam's client SDKs."
+tags: ["cloud", "sdk", "viam-server", "networking", "apis", "robot api", "cloud management"]
+---
+
+The cloud app API allows you to [manage your robot fleet](/manage/fleet/) with code instead of with the graphical interface of the [Viam app](https://app.viam.com/).
+Functionality includes:
+
+- create and manage organizations, locations, and individual robots
+- manage permissions and authorization
+- create and manage fragments
+
+{{% alert title="Support Notice" color="note" %}}
+
+Cloud app API methods are only available in the Python SDK.
+
+{{% /alert %}}
+
+## Establish a connection
+
+To use the Viam cloud app API, you first need to instantiate a [`ViamClient`](https://python.viam.dev/autoapi/viam/app/viam_client/index.html#viam.app.viam_client.ViamClient) and then instantiate an [`AppClient`](https://python.viam.dev/autoapi/viam/app/app_client/index.html#viam.app.app_client.AppClient).
+See the following example for reference.
+You can find the location secret and the URL of your robot on the **Code sample** tab in [Viam app](https://app.viam.com/):
+
+```python {class="line-numbers linkable-line-numbers"}
+from viam.rpc.dial import DialOptions, Credentials
+from viam.app.viam_client import ViamClient
+
+async def connect() -> ViamClient:
+    dial_options = DialOptions(
+        auth_entity='mrroboto.this_is_just_an_example.viam.cloud',  # The URL of your robot.
+        credentials=Credentials(
+            type='robot-location-secret',
+            payload='YOUR ROBOT LOCATION SECRET' # The location secret of your robot
+        )
+    )
+    return await ViamClient.create_from_dial_options(dial_options)
+
+async def main():
+
+  # Make a ViamClient
+  viam_client = await connect()
+  # Instantiate an AppClient called "cloud" to run the cloud app API methods on
+  cloud = await viam_client.app_client()
+```
+
+Once you have instantiated an `AppClient`, you can run methods against the `AppClient` object (named `cloud` in the examples) according to the examples for each method below.
```

Comment:
```suggestion
Once you have instantiated an `AppClient`, you can run the following [API methods](#api) against the `AppClient` object (named `cloud` in the examples).
```

Reply:
this sentence should probably just go under the API heading

Reply:
That'll make the API section not match the API sections for other docs. Do we care?